plugins/git/commands

```diff
@@ -20,11 +20,17 @@ git_build_app_repo() {
   git submodule update --init --recursive &> /dev/null
   find -name .git -prune -exec rm -rf {} \; > /dev/null
 
-  if [[ -f Dockerfile ]] && [[ "$([[ -f .env ]] && grep -q BUILDPACK_URL .env; echo $?)" != "0" ]] && [[ ! -f ".buildpacks" ]]; then
-    dokku receive "$APP" "dockerfile" "$TMP_WORK_DIR" | sed -u "s/^/"$'\e[1G'"/"
-  else
-    dokku receive "$APP" "herokuish" "$TMP_WORK_DIR" | sed -u "s/^/"$'\e[1G'"/"
+  if [[ "$([[ -f .env ]] && grep -q BUILDPACK_URL .env; echo $?)" != "0" ]] && [[ ! -f ".buildpacks" ]]; then
+    if [[ -f Dockerfile ]] && [[ -f docker-compose.yml ]]; then
```

Comment:
This still allows that. If a user has a `BUILDPACK_URL` set *or* `.buildpacks` file, it will use the buildpacks for deployment.

Maybe we should introduce a method to force a specific type of deployment instead of interpolating this?